bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v2/PageImpl.java

```diff
@@ -116,10 +119,25 @@ private String getProxyPath(ClientLibrary lib) {
         return path;
     }
 
+
+    protected void populateHeadClientlibCategories() {
```

Comment:
@znikolovski - agree, what would your expectation be here for a clash? I think I would expect the JS for a clientlib defined in both fields to be added to the head but not the body.

Reply:
@richardhand that's what I was thinking - I almost consider this an "explicit" request to put the clientlib in the head so it should win out.

Wouldn't this also mean that the populateClientlibCategories() method should also be overriden to implement the check?

Reply:
@richardhand I feel what you're saying is right and makes a lot of sense as loading the js in the head is the better option. In that case we won't be needing the field for loading the js in the body. Also, we'll need to make adjustments to the HTL call since currently it's calling the clientlibs in the body element as well.

Reply:
It's not that loading in the head is a better option, rather we want to provide the option to "explicitly" do so. We still need a general clientlib definition (both CSS and JS) with body end being the default.